CONTRIBUTING.md

```diff
@@ -103,6 +103,18 @@ which can be configured with Google style.
 To do so, after having installed the plugin, go to
 `Atom -> Preferences... -> Packages -> clang-format -> Settings` and type `google` in the `Style` field.
 
+#### Python coding style
+
+Python code should follow [PEP 8](https://TODO).
+For consistency, we use the [black](https://TODO) code formatter, which you can install in you Python environment using `pip install black`.
+If you add or change Python code, you should run black via
```

Comment:
I think it's "you can" rather "you should", because for example I've never used `bash` to run `black` in my entire life and I've been using `black` extensively for the last ~9 months.